distr_tracing/distr_tracing_rn/distr-tracing-rn-2-5.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: ASSEMBLY
+[id="distributed-tracing-rn-2-5"]
+= Release notes for {DTProductName} 2.5
+include::_attributes/common-attributes.adoc[]
+:context: distributed-tracing-rn-2-5
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
```

Comment:
[Anchoring in assembly files](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-assembly-files)
Global comment: Missing anchor IDs throughout the assembly file

Reply:
Done. Thank you.